plugins/importexport/pubmed/PubMedExportPlugin.php

```diff
@@ -191,14 +191,22 @@ public function exportIssues($issueIds, $context, $user)
         $pubmedExportFilters = $filterDao->getObjectsByGroup('article=>pubmed-xml');
         assert(count($pubmedExportFilters) == 1); // Assert only a single serialization filter
         $exportFilter = array_shift($pubmedExportFilters);
-        $submissions = Repo::submission()
-            ->getCollector()
-            ->filterByContextIds([$context->getId()])
-            ->filterByIssueIds($issueIds)
-            ->getMany();
+        $input = [];
+
+        foreach ($issueIds as $issueId) {
+            $sections = Repo::section()->getByIssueId($issueId);
+            foreach ($sections as $section) {
+                $submissionsInSections = [];
+                $submissionsInSections[] = Repo::submission()->getInSections($issueId, $context->getId());
+                foreach ($submissionsInSections as $articles) {
+                    foreach ($articles[$section->getId()]['articles'] as $article) {
+                        $input[] = $article;
```

Comment:
If you have `$submissionsInSections = Repo::submission()->getInSections($issueId, $context->getId());`  as said above then maybe you can only use
`$input = array_merge($input, $submissionsInSections[$section->getId()]);`